docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -1,27 +1,74 @@
 [[alerts-ui-manage]]
 [role="xpack"]
-== Managing detection alerts
+== Manage detection alerts
 
 The Detections page displays all <<detection-alert-def, detection alerts>>.
-From the Alerts table, you can change an alert's status, and start
+From the Alerts table, you can filter alerts, change an alert's status, and start
 investigating and analyzing alerts in Timeline.
 
 TIP: From Timeline, you can <<cases-ui-open, create cases>> to track issues and
 share information with colleagues.
 
-To view detection alerts created by a specific rule, you can:
+[float]
+[[detection-view-and-filter-alerts]]
+=== View and filter detection alerts
+The Detections page offers a variety of ways for you to organize and triage detection alerts as you investigate suspicious events. You can:
 
 * Filter for a specific rule in the KQL bar (for example,
 `signal.rule.name :"SSH (Secure Shell) from the Internet"`).
-* View detection alerts in the *Rule details* page (click
-*Manage detection rules* -> rule name in the *All rules* table).
 
 NOTE: KQL autocomplete for `.siem-signals-*` indices is available on the
 *Detections* and *Rule details* pages, and in Timeline when either `All` or
 `Detection alerts` is selected.
 
-TIP: Use the icons in the upper left corner of the Alerts table to customize
-displayed columns and row renderers, and view the table in full screen mode.
+* Use the date and time filter to select a time range that you’re interested in exploring. By default, this filter is set to search through the last 24 hours.
+* View detection alerts in the *Rule details* page (click
+*Manage detection rules* -> rule name in the *All rules* table).
+* Use the *Stack by* dropdown in the Detection alert trend histogram to select specific parameters for which to visualize the individual counts. For example, if you choose to stack by `signal.rule.name`, the histogram will display the total counts by alert name.
+* Augment the results shown in the Alerts table to include alerts from building blocks or only show alerts from indicator match rules. By default, building block alerts are excluded from the Alerts table. Opting to include them expands the results and displays building block alerts alongside regular detection alerts.
```

Comment:
@rylnd @shimonmodi this is an interesting one. So we are filtering the alerts on signal.rule.threat_mapping - which I believe exists in all indicator match rules. Are we anticipating a future scenario in which threat match rules is a sub-category of indicator match rules, where `threat_mapping` exists only on indicator matches of threat type? Unless I am mistaken, we are using the two concepts of indicator match and threat match interchangeably at the moment, which could create confusion for the end users.